sklearn/metrics/cluster/_supervised.py

```diff
@@ -689,19 +689,15 @@ def v_measure_score(labels_true, labels_pred, *, beta=1.0):
       1.0
 
     Labelings that assign all classes members to the same clusters
-    are complete be not homogeneous, hence penalized::
+    are complete but not homogeneous, hence penalized::
 
       >>> print("%.6f" % v_measure_score([0, 0, 1, 2], [0, 0, 1, 1]))
       0.8...
-      >>> print("%.6f" % v_measure_score([0, 1, 2, 3], [0, 0, 1, 1]))
-      0.66...
```

Comment:
This is similar to the completeness case below. In this case, the `homogeneity_score` decreased which result sin the `v_measure_score` going down.